src/layouts/partial-cards.html

```diff
@@ -38,7 +38,7 @@
     </style>
     <app-route
       route="{{route}}"
-      pattern="/states/:view"
+      pattern="/:view"
```

Comment:
This will cause the problem that if you go to `/automation/edit/bla`, it will trigger this `app-route` in the states panel. You can circumvent this by checking if `panelVisible`.

This problem only occurs in states, as that panel is never unloaded (the others are)